lib/datasource/npm/index.spec.ts

```diff
@@ -12,8 +11,6 @@ const datasource = NpmDatasource.id;
 jest.mock('registry-auth-token');
```

Comment:
```suggestion
```

No longer required? Can we also remove `registry-auth-token` dependency from `package.json` now?

Reply:
It's still used for URLs, but will eventually be removed